app-sso/getting-started/appsso-overview.hbs.md

```diff
@@ -17,6 +17,8 @@ After completing these steps, you can proceed with
 You must install AppSSO on your Tanzu Application Platform cluster.
 For more information, see [Install AppSSO](../how-to-guides/platform-operators/installation.hbs.md).
 
+You must have the [Tanzu CLI](/install-tanzu-cli.hbs.md) installed on your machine and be connected to a Tanzu cluster.
```

Comment:
**[question]:** This links to root `/`. Is that intended?

Reply:
Yes, it is intended! I hope it works this way, but that was the only way the IDE wasn't complaining about the path...